xml/adm_sudo.xml

```diff
@@ -555,7 +539,7 @@ User_List Host_List = [(User_List)] [NOPASSWD:|PASSWD:] Cmnd_List
      </para>
      <para>
       Depending on whether you want users to re-enter their passwords,
-      uncomment the specific line in <filename>/etc/sudoers</filename> and
+      uncomment the appropriate line in <filename>/etc/sudoers</filename> and
       comment out the default rule.
      </para>
```

Comment:
I'm not sure if you should change `/etc/sudoers` directly. In a way, it contradicts your own recommendation. :wink: 
Couldn't you add these two comments into the newly created `/etc/sudoers.d/userpw` file?
I think, that would avoid to edit the main file and you can see it as a holistic solution.